src/common/l10n.c

```diff
@@ -189,7 +190,7 @@ static void get_language_names(GList *languages)
     goto end;
   }
 
-  char *saved_locale = strdup(setlocale(LC_ALL, NULL));
```

Comment:
Well, this is problem because strdup may return NULL in rare case when it cannot allocate memory... well... It actually may crash if setlocale returns null here, but it wouldn't since setlocale with null as param will return current already allocated locale.